tests/unit/test_presenter.py

```diff
@@ -16,12 +16,10 @@ def setUp(self) -> None:
         self.user_id = 0
 
     def test_with_bad_key(self) -> None:
```

Comment:
Please add a test for two presenters with different auth modes

Reply:
done